frontend/app/src/components/eth-address-book-manager/EthAddressBookManagement.vue

```diff
@@ -112,129 +109,102 @@ import {
   EthNamesPayload
 } from '@/types/eth-names';
 
-export default defineComponent({
-  name: 'EthAddressBookManagement',
-  components: {
-    EthNamesHint,
-    EthAddressBookForm,
-    EthAddressBookTable,
-    BigDialog
-  },
-  setup() {
-    const search = ref<string>('');
-    const pendingSearch = ref<string>('');
-
-    const tab = ref<number>(0);
-    const locations: EthAddressBookLocation[] = ['global', 'private'];
-    const { tc } = useI18n();
-
-    const location = computed<EthAddressBookLocation>(() => {
-      return locations[get(tab)];
-    });
-
-    const emptyForm: () => EthNamesPayload = () => ({
-      location: get(location),
-      address: '',
-      name: ''
-    });
+const search = ref<string>('');
+const pendingSearch = ref<string>('');
 
-    const openForm = (item: EthNamesEntry | null = null) => {
-      set(editMode, !!item);
-      if (item) {
-        set(form, {
-          ...item,
-          location: get(location)
-        });
-      } else {
-        const newForm = emptyForm();
-        set(form, {
-          ...newForm
-        });
-      }
-      set(showForm, true);
-    };
+const tab = ref<number>(0);
+const locations: EthAddressBookLocation[] = ['global', 'private'];
+const { tc } = useI18n();
 
-    const hideForm = function () {
-      set(showForm, false);
-      set(form, emptyForm());
-    };
-
-    const valid = ref<boolean>(false);
-    const showForm = ref(false);
-    const editMode = ref<boolean>(false);
-    const form = ref<EthNamesPayload>(emptyForm());
-
-    const {
-      isPending: isTimeoutPending,
-      start,
-      stop
-    } = useTimeoutFn(
-      () => {
-        set(search, get(pendingSearch));
-      },
-      600,
-      { immediate: false }
-    );
-
-    const onSearchTermChange = (term: string) => {
-      set(pendingSearch, term);
-      if (get(isTimeoutPending)) {
-        stop();
-      }
-      start();
-    };
+const location = computed<EthAddressBookLocation>(() => {
```

Comment:
Check the warning above, maybe in the template's v-for we can change the name from location to something else